src/controllers/directReport.controller.js

```diff
@@ -60,9 +60,10 @@ export const approve_reject_TravelRequest = async (req, res, next) => {
                   };
                   
                const notification = await models.Notification.create(newNotificantion);
-               pusher.trigger('bare-foot-normad', 'notification', notification);
+              //  pusher.trigger('bare-foot-normad', 'notification', notification);
```

Comment:
@Noel2002 I'd like to know any reason why you've commented this line yet this PR is about sending & receiving push notifications.

Reply:
It is due to the fact that I used different approach and it was no longer useful.